src/main/java/org/folio/circulation/infrastructure/storage/loans/LoanRepository.java

```diff
@@ -224,6 +225,16 @@ else if(loans.getTotalRecords() == 1) {
       .thenComposeAsync(loans -> itemRepository.fetchItemsFor(loans, Loan::withItem));
   }
 
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findByItemIds(
+    Collection<String> itemIds) {
+
+    Result<CqlQuery> statusQuery = exactMatch("itemStatus", IN_TRANSIT.getValue());
```

Comment:
Use already defined constant ITEM_STATUS

Reply:
Have you merged the PR?

Reply:
Found, updated, thanks